stable/rabbitmq/values-production.yaml

```diff
@@ -139,6 +139,11 @@ service:
   annotations:
     # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
+  #  - 10.10.10.0/24
```

Comment:
Can you add a line only with `##` between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```